RecordShow.java

```diff
@@ -0,0 +1,113 @@
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.util.Scanner;
+
+class RecordShow {
+    // start Recoding after inserting programme name and time id
+    int user_id;
+
+    void recordShow(Scanner inp, int user_id) throws Exception {
+        user_id = this.user_id;
+        PreparedStatement pst;
+        int rs;
+        ResultSet resultSet;
+        SQLQueries sql = new SQLQueries();
+
+        String program_routine = "select * from program_routine join channel on program_routine.channel_id=channel.channel_id join  time_slot on program_routine.time_slot_id=time_slot.time_slot_id ";
+        pst = sql.con.prepareStatement(program_routine);
+        resultSet = pst.executeQuery();
+        while (resultSet.next()) {
+            System.out.println(resultSet.getInt("program_id") + "\t"
+                    + resultSet.getString("program_name") + "\t                  " + resultSet.getTime("start_time")
+                    + "-"
+                    + resultSet.getTime("end_time"));
+        }
+
```

Comment:
How will the user know the program ID???
There are no functionality to know that
Instead ask which channel you want to record from and the time

Reply:
here i  have display every program_rouitne with id . and in this i found a mistake as   customer or user have different packages so the can record as it but how to connect idk just now every channenel is available for user